b2/src/main/java/org/jclouds/b2/features/AuthorizationApi.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.labs.b2.features;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+import javax.inject.Named;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+
+import org.jclouds.labs.b2.domain.Authorization;
+import org.jclouds.http.HttpException;
+import org.jclouds.http.HttpRequest;
+import org.jclouds.http.HttpRequestFilter;
+import org.jclouds.rest.annotations.Headers;
+import org.jclouds.rest.annotations.RequestFilters;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.BaseEncoding;
+
+public interface AuthorizationApi {
+   @Named("b2_authorize_account")
+   @GET
+   @Path("/b2api/v1/b2_authorize_account")
+   @Headers(keys = "Authorization", values = "{accountId}:{applicationKey}")
+   @RequestFilters(Base64EncodeAuthorization.class)
```

Comment:
Remove the `@Headers` annotation and use the existing filter: `@RequestFilter(BasicAuthentication.class)`.
Also, since the accountId is the "identity", you could create a class that extends the jclouds `MapBinder` (the one that binds the values tot he request the request) and has the Credentials injected. The mapbinder could then update the path with the identity, and you remove the need for the first "accountId" parameter.